### DISCUSSION TITLE
Github action is not working

### DISCUSSION THREAD

**Discussion by Salman-18** [MuhammadNizamani/nizchain#8] [Q&A] [answered]
Did you notice that when I created a pull request against the original repo, the GitHub Actions didn't run? I'm not sure why this happened. Any ideas? @MuhammadNizamani

**Comment by MuhammadNizamani** [comment]
I have checked in the action tab and it is showing that action did run latest PR

![Screenshot_2024-06-02-22-20-29-70_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/MuhammadNizamani/nizchain/assets/49721249/3ae7e3b2-15e2-49fe-9b66-fef834124795)

**Reply by MuhammadNizamani** [reply, marked as answer]
@Salman-18  please mark this as answer
